tox.ini

```diff
@@ -17,6 +17,7 @@ deps =
 	pycodestyle
 	pylint
 commands =
-	pycodestyle calc/__init__.py calc/tests/__init__.py
-	pylint calc/tests/__init__.py
+	pycodestyle calc/__init__.py
+	pycodestyle symmath/__init__.py
+	pylint tests/__init__.py
```

Comment:
Check out the logs of the `quality` PR check -- this isn't being run.

Reply:
Quality tests are running